CHANGES.rst

```diff
@@ -1,6 +1,8 @@
 Upcoming version
 ================
 
+- Introduce FSSeries class for Rohde&Schwarz instruments such as FSL (previously stand-alone class), FSW and others from the same product series (@jnnskls, #1156)
+  Previously, the instrument class FSL could be imported from :code:`pymeasure/instruments/rohdeschwarz/fsl.py`, which can now be imported from :code:`pymeasure/instruments/rohdeschwarz/fsseries.py` alongside the base class FSSeries and FSW.
```

Comment:
Please open a Deprecated section, where you state, that it is deprecated to use the fsl file.